### PR TITLE
[txn] Add check for prewrite ttl for pessimistic txn

### PR DESCRIPTION
**Pull request by cfzjywxk**
- Add check for prewrite TTL for pessimistic transactions like tikv
- Refactor some test utilities, increase the `raftstore` coverage(11% -> 27%)

### CODE REVIEW

**Comment by cfzjywxk** [comment]
@coocood @bobotu  PTAL

File: tikv/mvcc_test.go [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+	defer CleanTestStore(store)
+
+	reqCtx := &requestCtx{
+		regCtx: &regionCtx{
+			latches: make(map[uint64]*sync.WaitGroup),
+		},
+		svr: store.Svr,
+	}
+
+	// Prewrite key1 with smaller lock ttl
```
Comment (coocood) [inline tikv/mvcc_test.go]: This comment seems unrelated.

**Comment by coocood** [comment]
LGTM
